user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_components.md

```diff
@@ -0,0 +1,189 @@
+---
+uid: VSAT_components
+---
+
+# EPM VSAT components
+
+## Connectors
+
+### Collector connectors
+
+Collector connectors interface with the main data sources (iDirect Evolution, iDirect Dialog, SES Skala, Intelsat Flex, etc.). For example, we use collector connectors to poll an iDirect Evolution NMS and obtain the necessary information needed to present remote information data throughout the EPM topology.
+
+While these connectors ship out with the solution packages, they need to be contracted separately as necessary.
+
+#### iDirect Evolution
+
+This is the collector connector required for the iDirect Evolution Platform. This collector was primarily tested on version 4.1 or later.
+
+> [!TIP]
+> For more information, see [Verizon iDirect Evolution Platform Collector](https://catalog.dataminer.services/result/driver/5827).
+
+#### iDirect Dialog Platform VSAT
+
+This is the collector connector required for the iDirect (formerly Newtec) Dialog Platform with Time Series Database (TSDB) integration.
+
+> [!TIP]
+> For more information, see [Newtec Dialog Platform VSAT](https://catalog.dataminer.services/result/driver/7342).
+
+#### CMDB
+
+This is the collector connector that helps form relationships between various collectors and technology that are traditionally unrelated. It can provide additional business details of the customer to enhance the metrics and KPIs needed that are not obtained by the NMS collectors alone.
+
+### EPM connectors
+
+EPM connectors constitute the core of the EPM Solution. They carry out the necessary logic to aggregate and display the information of the different topology levels. EPM connectors rely on collectors and system connectors to feed information into the EPM engine.
+
+EPM connectors ship with the EPM Solution package.
+
+## Peripherals
+
+Peripherals are optional but not required as part of a standard deployment.
+
+### Sun outage
+
+DataMiner predicts, detects falls, and offers the current state details of each remote in the VSAT system referring to the sun outage events that might affect the transmission. These events normally occur twice a year when the day arc takes place directly behind the line of sight of a satellite and an earth station.
+
+### Weather
+
+DataMiner has flexibility of integration with any weather forecast service as it considers this variable a key factor capable of degrading and temporarily interrupting the signal delivery via VSAT. Operators receive automated detailed descriptions of the physical and remote falls that were correlated to weather conditions.
```

Comment:
What do you mean with falls here and in the description for PLM below? I'm not sure if this is clear enough.